packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -213,32 +89,66 @@ const ErrorFormTooltip = ({
 
+  const isVertical =
+    attrs.popper &&
+    (attrs.popper["data-popper-placement"] === "top-start" ||
+      attrs.popper["data-popper-placement"] === "top-end")
+      ? "bottom"
+      : "top";
+
+  const cssVars = {
+    "--error-form-tooltip-position": popper.position,
+    "--error-form-tooltip-top": popper.top,
+    "--error-form-tooltip-left": popper.left,
+    "--error-form-tooltip-right": popper.right,
+    "--error-form-tooltip-bottom": popper.bottom,
+    "--error-form-tooltip-transform": popper.transform,
+  };
+
   return (
-    <StyledFormFeedbackTooltip
+    <div
       id={id}
       ref={tooltipRef}
-      shown={shown}
-      isHelp={isHelp}
-      data-test={dataTest}
       aria-live="polite"
-      inlineLabel={inlineLabel}
-      placement={attrs.popper && attrs.popper["data-popper-placement"]}
-      position={popper.position}
-      top={popper.top}
-      left={popper.left}
-      right={popper.right}
-      bottom={popper.bottom}
-      transform={popper.transform}
+      data-test={dataTest}
+      className={cx(
+        "flex justify-between overflow-visible",
+        "rounded-large py-xs px-sm z-10 box-border",
+        "max-h-none w-[min(calc(100%-20px),_100vw)]",
+        isHelp ? "pe-sm bg-blue-normal" : "bg-red-normal",
+        shown ? "visible opacity-100" : "invisible opacity-0",
+        "duration-fast transition-[opacity,visibility] ease-in-out",
+        "bottom-[var(--error-form-tooltip-bottom)] left-[var(--error-form-tooltip-left)] right-[var(--error-form-tooltip-right)] top-[var(--error-form-tooltip-top)] [position:var(--error-form-tooltip-position)] [transform:var(--error-form-tooltip-transform)]",
+        "lm:w-auto tb:rounded-normal",
+        "[&>img]:max-w-full",
+      )}
+      style={cssVars as React.CSSProperties}
     >
-      <StyledArrow
-        isHelp={isHelp}
+      <div
+        className={cx(
+          "start-xs de:start-0 absolute",
+          isVertical ? "bottom-xxxs" : "top-xxxs",
+          inlineLabel && "rtl:start-0",
+          isHelp ? "before:bg-blue-normal" : "before:bg-red-normal",
+          "before:h-xs before:w-xs before:absolute before:-translate-x-1/2 before:-translate-y-1/2 before:rotate-45",
+        )}
         ref={setArrowRef}
-        inlineLabel={inlineLabel}
-        placement={attrs.popper && attrs.popper["data-popper-placement"]}
       />
-      <StyledTooltipContent ref={contentRef}>{children}</StyledTooltipContent>
+      <div
+        className={cx(
+          "font-base text-normal leading-small text-white-normal flex h-full items-center font-normal",
+          "[&_.orbit-text]:text-white-normal [&_.orbit-text]:text-normal hover:[&_.orbit-text]:text-white-normal focus:[&_.orbit-text]:text-white-normal [&_.orbit-text]:font-normal",
+          "[&_.orbit-list-item]:text-white-normal [&_.orbit-list-item]:text-normal hover:[&_.orbit-list-item]:text-white-normal focus:[&_.orbit-list-item]:text-white-normal [&_.orbit-list-item]:font-normal",
```

Comment:
do we need these focus and hover classes if the color is not changing?

Reply:
besides that nothing. LGTM and can be merged :) @oreqizer I though we were waiting for you approval on that also :)

Reply:
I believe it is here to enforce the color. If we use the Text inside, the default colors of the text are given by classes that will always be overwritten by this. It's true, we don't seem to have styles on hover which means we could probably remove it. I would personally not risk it at this moment tbh. But fair question